packages/idos-sdk-js/src/lib/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { decodeBase58, toBeHex } from "ethers";
+import * as nearAPI from "near-api-js";
+
+export async function getNearFullAccessPublicKey(namedAddress: string) {
+  const { connect } = nearAPI;
+  const connectionConfig = {
+    networkId: import.meta.env.VITE_IDOS_NEAR_DEFAULT_NETWORK,
+    nodeUrl: import.meta.env.VITE_IDOS_NEAR_DEFAULT_RPC_URL
+  };
```

Comment:
Careful. This is not following what the user specifies on the constructor. That's going to be really confusing.

Reply:
Very much so. We'll have to do that sweeping change later. Let's go. :rocket: